### ISSUE TITLE
UX issue in notepad example on network where ENS isn't supported

### ISSUE THREAD

**Issue by LuaxY** [spruceid/siwe#13]
If we try to log in with MetaMask on notepad example on network that don't support ENS nothing happen on the UI and seem blocked, but the following error is throw in the console:

```js
Uncaught (in promise) Error: network does not support ENS (operation="ENS", network="maticmum", code=UNSUPPORTED_OPERATION, version=providers/5.5.0)
    at Logger.makeError (index.js:185)
    at Logger.throwError (index.js:194)
    at Web3Provider.<anonymous> (base-provider.js:1497)
    at Generator.next (<anonymous>)
    at fulfilled (base-provider.js:5)
```

It should be a good idea, in terms of user experience, to handle the error by ignoring ENS or informing the user that ENS isn't supported in the selected network on MetaMask.

**Comment by w4ll3** [comment]
Great catch. I've opened a PR to resolve it, should be merged soon.

**Comment by w4ll3** [comment]
Closed via #15
